### ISSUE TITLE
Select random data while train and test?Is that suitable?

### ISSUE THREAD

**Issue by Zhangang1999** [danieltan07/dagmm#23]
I am not sure about that.Because will it lead to overfitting?

**Comment by chiachen-chang** [comment]
Yes, it's suitable to use the KDD dataset for unsupervised learning tasks. The training set includes only positive instances, while the testing set consists of both positive and negative ones. It's important to note that all the positive instances in the test set should be included in the training set, making this set-up appropriate for unsupervised learning.
